.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+/dasch-service-platform-repo.iml
+/modules.xml
+/vcs.xml
```

Comment:
do we want `.idea` stuff in the repo?

Reply:
idk really, this is part of my IDE battle 😓

Reply:
you should probably remove it. editor configurations don't belong in the repository.

Reply:
removed in a6606a5